src/tests/workflows/recipes/components/FilterObjectForm.test.js

```diff
@@ -0,0 +1,509 @@
+import { List, fromJS } from 'immutable';
+import { render, fireEvent } from 'react-testing-library';
+import fetchMock from 'fetch-mock';
+import { NORMANDY_ADMIN_API_ROOT_URL } from 'console/settings';
+import FilterObjectForm, {
+  serializeFilterObjectToMap,
+  deserializeFilterObjectToList,
+  InputsWidget,
+  rewriteSamplingInput,
+  SamplingInput,
+  VersionsInput,
+  parseIntOrNull,
+} from 'console/workflows/recipes/components/FilterObjectForm';
+import { createForm } from 'console/utils/forms';
+import { wrapMockStore } from 'console/tests/mockStore';
+
+describe('<FilterObjectForm>', () => {
+  it('render editing form with an empty existing filter object', () => {
```

Comment:
The testing style used throughout the rest of the code reads the whole line as a sentence, like "It render editing form...", notably including the "it" at the beginning. "It should..." is a common phrasing. The use of the tense "render" here makes that sound really awkward.

Reply:
I've now rewritten them all to read more pleasantly. (not pushed yet)